### PR TITLE
Hide various links for orcharhino builds

### PR DESCRIPTION
**Pull request by maximiliankolb**
Please review commit by commit.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)

### CODE REVIEW

File: guides/common/modules/proc_registering-hosts.adoc [hunk 1]
```diff
@@ -16,13 +16,19 @@ endif::[]
...
+ifdef::satellite,orcharhino[]
 * {ProjectServer}, any {SmartProxyServer}s, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
 * The daemon *rhsmcertd* must be running on the hosts.
 * An activation key must be available for the host.
 For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
 * Subscription Manager must be version 1.10 or later.
+endif::[]
+ifdef::satellite[]
 The package is available in the standard {RHEL} repository.
 endif::[]
```
Comment (adamlazik1) [inline guides/common/modules/proc_registering-hosts.adoc:33]: These don't appear to be ordered so I am for simplifying
```suggestion
* You must have root privileges on the host that you want to register.
ifdef::satellite,orcharhino[]
* You must have an activation key created.
* {ProjectServer}, any {SmartProxyServer}s, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
* The daemon *rhsmcertd* must be running on the hosts.
* An activation key must be available for the host.
For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
* Subscription Manager must be version 1.10 or later.
endif::[]
ifdef::satellite[]
The package is available in the standard {RHEL} repository.
* Optional: If you want to register hosts to Red{nbsp}Hat Insights, you must synchronize the `{RepoRHEL7Server}` repository and make it available in the activation key that you use.
This is required to install the `insights-client` package on hosts.
endif::[]
```

**Review by adamlazik1 — COMMENTED** [review]
LGTM, only one nitpick.

File: guides/common/modules/proc_registering-hosts.adoc [hunk 2]
```diff
@@ -163,7 +163,7 @@ endif::[]
 . Copy the generated `curl` command.
 . On the hosts that you want to register, run the `curl` command as `root`.
 
-ifdef::katello,satellite,orcharhino[]
+ifdef::katello,satellite[]
 [NOTE]
 ====
 For {RHEL} 6.3 hosts, the release version defaults to {RHEL} 6 Server and needs to be pointed to the 6.3 repository:
```
Comment (ekohl) [inline guides/common/modules/proc_registering-hosts.adoc:169]: I think by now RHEL 6.3 is unsupported, even from Red Hat's point of view. Only the last RHEL 6 version (6.10 IIRC) can still be supported. I wonder what the point of this note is and if we shouldn't drop it altogether.

File: guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc [hunk 3]
```diff
@@ -63,7 +63,7 @@ endif::[]
 ----
 +
 Replace _OTP_ with the one-time password provided by the {FreeIPA} administrator.
-ifndef::satellite[]
+ifndef::satellite,orcharhino[]
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc:66]: @evgeni should this be
```suggestion
ifdef::foreman-deb[]
```

File: guides/common/modules/proc_registering-hosts.adoc [hunk 4]
```diff
@@ -16,13 +16,19 @@ endif::[]
...
+ifdef::satellite[]
 * Optional: If you want to register hosts to Red{nbsp}Hat Insights, you must synchronize the `{RepoRHEL7Server}` repository and make it available in the activation key that you use.
 This is required to install the `insights-client` package on hosts.
+endif::[]
+ifdef::satellite,orcharhino[]
 * {ProjectServer}, any {SmartProxyServer}s, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
 * The daemon *rhsmcertd* must be running on the hosts.
 * An activation key must be available for the host.
 For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
 * Subscription Manager must be version 1.10 or later.
```
Comment (ekohl) [inline guides/common/modules/proc_registering-hosts.adoc:29]: I checked and CentOS 7 ships version 1.24.51. https://vault.centos.org/centos/6/contrib/x86_64/Packages/ shows that CentOS 6 contrib had version 1.20 so I wonder if listing this requirement even makes sense nowadays. I don't have a good overview of the various supported RHEL EUS versions and the subman version, but I think dropping it is fairly safe.

File: guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc (hunk 3 above)
Comment (evgeni) [reply guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc:66]: Sounds reasonable, and then we should drop the "If …" part - the same applies to Ubuntu too.

File: guides/common/modules/proc_registering-hosts.adoc (hunk 4 above)
Comment (evgeni) [reply guides/common/modules/proc_registering-hosts.adoc:29]: I think only 6.10 is still under any kind of support, so yeah

File: guides/common/modules/proc_registering-hosts.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_registering-hosts.adoc:33]: Applied.

**Comment by maximiliankolb** [comment]
Applied all feedback; ready for re-review.

File: guides/common/modules/proc_importing-kickstart-repositories.adoc [hunk 5]
```diff
@@ -4,6 +4,7 @@
 Use this procedure to import Kickstart repositories for {RHEL}{nbsp}{el-major-version}.
 
 .Procedure
+ifndef::orcharhino[]
```
Comment (ekohl) [inline guides/common/modules/proc_importing-kickstart-repositories.adoc:7]: I do wonder if it wouldn't be better if you do provide more detailed instructions for RHEL customers, but I'll leave that to you.

File: guides/common/modules/proc_registering-hosts.adoc [hunk 6]
```diff
@@ -16,14 +16,14 @@ endif::[]
...
-This is required to install the `insights-client` package on hosts.
 * {ProjectServer}, any {SmartProxyServer}s, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
 * The daemon *rhsmcertd* must be running on the hosts.
 * An activation key must be available for the host.
 For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
-* Subscription Manager must be version 1.10 or later.
-The package is available in the standard {RHEL} repository.
+endif::[]
+ifdef::satellite[]
+* Optional: If you want to register hosts to Red{nbsp}Hat Insights, you must synchronize the `{RepoRHEL7Server}` repository and make it available in the activation key that you use.
```
Comment (ekohl) [inline guides/common/modules/proc_registering-hosts.adoc:25]: Unrelated to your PR, but I wonder why it mentions the RHEL 7 repo. I'd think it's actually the RHEL version that the client is running.

**Review by ekohl — APPROVED** [review]

File: guides/common/modules/proc_importing-kickstart-repositories.adoc (hunk 5 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_importing-kickstart-repositories.adoc:7]: Yes, I will add something after introducing the `red_hat_enterprise_linux` attribute later on. For now, This is good enough for all different OS.

context: in downstream, we replace `{RHEL}` with all supported OS for managed hosts.

File: guides/common/modules/proc_registering-hosts.adoc (hunk 6 above)
Comment (adamlazik1) [reply guides/common/modules/proc_registering-hosts.adoc:25]: I believe RepoRHEL8BaseOS and RepoRHEL8AppStream together replace the RepoRHEL7Server on RHEL 8.

Comment (maximiliankolb) [reply guides/common/modules/proc_registering-hosts.adoc]: Applied your suggestion in the last commit.

**Comment by maximiliankolb** [comment]
Rebased to master and added one more commit on top. Please re-review.

**Comment by maximiliankolb** [comment]
@adamlazik1 Is this OK for you?

**Review by adamlazik1 — APPROVED** [review]
Yes, LGTM! Thank you!

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   704a3c2c6..9e26439f7  3.6 -> 3.6
   6f25b5ba9..1beff0a53  3.5 -> 3.5
   847f4e7f2..92dcdc4d0  3.4 -> 3.4
   465dc2ff9..32d4649cb  3.3 -> 3.3

**Pull request merged**